resources/i18n/en.json

```diff
@@ -334,7 +334,7 @@
         "f.dateIssued.max": "End date",
         "f.subject": "Subject",
         "f.has_content_in_original_bundle": "Has files",
-        "f.entityType": "Entity Type"
+        "f.entityType": "Item Type"
```

Comment:
@artlowel this Pull Request already assumes the replacement Entities with Item Type, unfortunately this is still an open topic - https://github.com/DSpace/DSpace/pull/2300. I agree with the replacement, but I'm afraid because there isn't any decision yet, this changes might be reverted. But I'm ok with it.

Reply:
This change seems perfectly fine to me. The Marketing Working group will be discussing new terminology next week (all based on "Item") and I expect the term "Entities" will likely be dropped anyhow.